src/main/java/seedu/address/logic/parser/FilterCommandParser.java

```diff
@@ -2,42 +2,47 @@
 
 import static seedu.address.logic.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
 
-import java.util.Arrays;
+import java.util.stream.Stream;
 
 import seedu.address.logic.commands.FilterCommand;
 import seedu.address.logic.parser.exceptions.ParseException;
-import seedu.address.model.person.TagContainsKeywordsPredicate;
 
 /**
- * Parses input arguments and creates a new FilterCommand object
+ * Parses input arguments and creates a new FilterCommand object.
  */
 public class FilterCommandParser implements Parser<FilterCommand> {
 
-    private static final Prefix TAG_PREFIX = CliSyntax.PREFIX_TAG;
-
     /**
      * Parses the given {@code String} of arguments in the context of the FilterCommand
      * and returns a FilterCommand object for execution.
-     * @throws ParseException if the user input does not conform the expected format
+     * @throws ParseException if the user input does not conform the expected format.
      */
     public FilterCommand parse(String args) throws ParseException {
-        String trimmedArgs = args.trim();
-        if (trimmedArgs.isEmpty()) {
+        ArgumentMultimap argMultimap =
+                ArgumentTokenizer.tokenize(args.trim(), CliSyntax.PREFIX_NAME, CliSyntax.PREFIX_TAG,
```

Comment:
It is possible to import these prefixes, so that you do not need to retype the class name over again, can potentially shorten code. Maybe can consider changing?

Reply:
Yea was rushing can fastgame do good eye